server/controllers/restaurant.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require("mongoose");
+const Restaurant = require("../models/restaurant");
+
+//API for getting all restaurants
+exports.restaurant_all = (req, res, next) => {
+    Restaurant.find({}).exec((err, e) =>{
+        if(err){
+            res.send(err)
+            return
+        }
+        
+        if(e){
+            var restaurants = "";
+
+            //put each event detail into string
+            for(i = 0; i < e.length; i++){
+                restaurants += "PlaceId: " + e[i].placeId + "<br>\n" +
+                "name: " + e[i].name + "<br>\n" +
+                "rating: " + e[i].rating + "<br>\n" +
+                "Latitude: " + e[i].latitude + "<br>\n" +
+                "Longitude: " + e[i].longitude + "<br>\n" +
+                "Comment: "  + e[i].comment + "<br>\n" + 
+                "<br>\n"
+            }
+            
+            res.send(restaurants)
```

Comment:
Maybe can add status code for response. For example, `res.status(200).send(restaurants)`
Also, using json instead of send maybe better ahahaha